src/com/xilinx/rapidwright/interchange/DeviceResourcesWriter.java

```diff
@@ -49,14 +51,36 @@
     private static Enumerator<String> allStrings;
     private static Enumerator<String> allSiteTypes;
 
-    private static HashMap<TileTypeEnum,Tile> tileTypes; 
+    private static HashMap<TileTypeEnum,Tile> tileTypes;
     private static HashMap<SiteTypeEnum,Site> siteTypes;
-    
-    public static void populateEnumerations(Device device) {
+
+    public static void populateSiteEnumerations(SiteInst site_inst, Site site) {
```

Comment:
@litghost - Thank you for all the updates, looks good.  

Not a big issue, but its nice to match the camelCase variable naming convention.   

At your suggestion, I did try looking into code style checkers.  The [Google code style checker for Java](https://github.com/google/google-java-format) seemed popular, but it doesn't check/correct naming schemes/conventions.  Are there tools that enforce/report variable naming conventions (camelCase vs. underscore) in any language?   

Eventually, I need to update the coding style to disallow tabs and purge the codebase of them.

Reply:
Ya, I'll fix up the under_score.  My muscle memory is `_`, but I don't mind fixing it.  This is one place where either a linting pass to detect or even an auto-format if possible is handy.

Reply:
`_` fixed